test/test_metrics.py

```diff
@@ -38,6 +38,14 @@ def test_clear_metrics(self):
     self.assertIn("TensorToData", met.metrics_report())
     assert (len(met.metric_names()) > 0)
 
+  def test_tracing_time_metrics(self):
+    xla_device = xm.xla_device()
+    met.clear_all()
+    t1 = torch.tensor(156, device=xla_device)
+    t2 = t1 + 100
+    self.assertIn('LazyTracing', met.metric_names())
+    self.assertGreater(met.metric_data('LazyTracing')[0], 1)
```

Comment:
So the op counter will still be int? But adding another LazyTracing metrics?

Reply:
yea.